prog/postgres/postgres_resource_nexus.rb

```diff
@@ -183,6 +183,7 @@ def before_run
   end
 
   label def update_billing_records
+    hop_wait unless postgres_resource.project.billable
```

Comment:
I looked into this more and I think we should run `decr_update_billing_records` and `decr_initial_provisioning` before hopping.